tensorflow/python/platform/gfile.py

```diff
@@ -401,7 +401,7 @@ def Walk(top, topdown=1, onerror=None):
     # by lists of all its subdirectories and leaf files.
```

Comment:
I would definitely add to the documentation that Walk follows symlinks by default though.